manifests/scenario/poss.pp

```diff
@@ -1,38 +0,0 @@
-# **NOTE: THIS IS A [PRIVATE](https://github.com/puppetlabs/puppetlabs-stdlib#assert_private) CLASS**
```

Comment:
I need to add this in for the logic it contains, just not classify anything, as it's actually an api